src/app/shared/breakpoint.service.ts

```diff
@@ -36,17 +36,17 @@ export class BreakpointService {
 
   private setupBreakpoints() {
     const small$ = this.breakPointObserver.observe(['(max-width: 600px)']).pipe(
-      pluck<BreakpointState, boolean>('matches'),
+      map(x => x.matches),
```

Comment:
Let's use a proper name instead of `x`, e.g. `state` because it yields the breakpoint state if I am not mistaken.